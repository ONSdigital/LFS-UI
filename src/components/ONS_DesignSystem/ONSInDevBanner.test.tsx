import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import {ONSInDevBanner} from "./ONSInDevBanner";
import Adapter from 'enzyme-adapter-react-16';

describe("ONS In Dev Banner Test", () => {
    Enzyme.configure({ adapter: new Adapter() });

    it("should render correctly", () => expect(shallow(<ONSInDevBanner></ONSInDevBanner>).exists()).toEqual(true));
})