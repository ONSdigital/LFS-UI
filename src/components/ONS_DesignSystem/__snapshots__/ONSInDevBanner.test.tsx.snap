// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ONS In Dev Banner Test matches Snapshot 1`] = `ShallowWrapper {}`;
